src/tests/stdlib/memmove.c

```diff
@@ -0,0 +1,59 @@
+#include <embox/test.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+EMBOX_TEST_SUITE("Test suite for memmove() function");
+
+
+TEST_CASE("move full data")
+{
+    const    char    src[]="string";
+    char    dest[sizeof(src)+1];
+
+    test_assert_not_null(dest) ;
```

Comment:
excess cheking

Reply:
yes , to be very careful .... :) :) :+1: